apps/journeys-admin/src/components/Editor/Slider/Content/Canvas/QuickControls/MoveBlock/MoveBlock.spec.tsx

```
@@ -4,13 +4,15 @@ import { userEvent } from '@testing-library/user-event'
 
 import type { TreeBlock } from '@core/journeys/ui/block'
 import { CommandProvider } from '@core/journeys/ui/CommandProvider'
-import { EditorProvider } from '@core/journeys/ui/EditorProvider'
+import { ActiveSlide, EditorProvider } from '@core/journeys/ui/EditorProvider'
 
 import {
   BlockOrderUpdate,
   BlockOrderUpdateVariables
 } from '../../../../../../../../__generated__/BlockOrderUpdate'
 import { blockOrderUpdateMock } from '../../../../../../../libs/useBlockOrderUpdateMutation/useBlockOrderUpdateMutation.mock'
+import { TestEditorState } from '../../../../../../../libs/TestEditorState'
```

ESLint

[error] `../../../../../../../libs/TestEditorState` import should occur before import of `../../../../../../../libs/useBlockOrderUpdateMutation/useBlockOrderUpdateMutation.mock`

(import/order)